osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs

```diff
@@ -342,8 +342,8 @@ private void forceAspectRatio(float aspectRatio)
 
         private float currentAspectRatio => sizeX.Value / sizeY.Value;
 
-        private static int getHeight(float width, float aspectRatio) => (int)Math.Round(width / aspectRatio);
+        private static float getHeight(float width, float aspectRatio) => MathF.Round(width / aspectRatio);
 
-        private static int getWidth(float height, float aspectRatio) => (int)Math.Round(height * aspectRatio);
+        private static float getWidth(float height, float aspectRatio) => MathF.Round(height * aspectRatio);
```

Comment:
Would prefer the type here to stay `int` too, I think? BIt weird for me to change this return type given the fact that the returned number is a whole number anyway.

Reply:
That was a mistake, I didn't mean to round it. I changed it now.